Lib/concurrent/futures/process.py

```diff
@@ -389,10 +390,14 @@ def wait_result_broken_or_wakeup(self):
         elif wakeup_reader in ready:
             is_broken = False
 
+        failed = [p for p in self.processes.values() if p.sentinel in ready]
+        exitcodes = [p.exitcode for p in failed]
+        exitcodes = [signal.Signals(-c) if c < 0 else c for c in exitcodes]
```

Comment:
Thanks for commits. A little misleading in naming. Maybe you would change one of the `exitcodes` lists:
```python
        exitcodes = [p.exitcode for p in failed]
        signals = [signal.Signals(-c) for c in exitcodes if c < 0 else c]
```

Reply:
The final value of `exitcodes` contains both exit-code integers and `signal.Signal` values. Perhaps it should be renamed `exitcodes_and_signals`?

Reply:
In my opinion, it’s worth waiting for what other reviewers will say. You can also make a test to check `__cause__` value.